examples/pattern_matching_showcase/experiment.test.sh

```diff
@@ -1,2 +0,0 @@
-mpirun -n 3 music `readlink -f run.music` TEST_MODE
-
```

Comment:
I think those helper scripts should be kept local to the experiment they belong to. Instructions on how to run an experiment should always be close to the experiment itself.

I suppose the best approach is to let this helper script encode how to run the experiment (in test mode) and the test should simply execute the helper script. I think the long (relative) path to the showcase is not avoidable.